frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -554,6 +594,50 @@ export function ContainerInfo(props: ContainerInfoProps) {
     );
   }
 
+  function handlePortForward(namespace?: string, podname?: string, containerPort?: number) {
+    if (!podname || !namespace || !containerPort) {
+      return;
+    }
+    fetch(
+      `${helpers.getAppUrl()}/startportforward?cluster=${cluster}&namespace=${namespace}&pod=${podname}&targetPort=${containerPort}`,
+      {
+        method: 'GET',
+        headers: new Headers({ Authorization: `Bearer ${localStorage.getItem('token')}` }),
+      }
+    )
+      .then(response => response.json())
+      .then(data => {
+        setPortForward(data);
+        setIsCurrentPodPortForwarding(true);
+
+        const portForwards = localStorage.getItem('portforwards');
+        if (!portForwards) {
+          const portForwardsList = [];
+          portForwardsList.push(JSON.stringify(data));
+          return;
+        }
+        const parsedPortForwards = JSON.parse(portForwards);
+        parsedPortForwards.push(data);
+        localStorage.setItem('portforwards', JSON.stringify(parsedPortForwards));
+      })
+      .catch(error => {
+        console.log(error);
```

Comment:
We should show an error dialog to warn the user that it didn't work.

Reply:
updated